ci/requirements-3.5.run

```diff
@@ -12,6 +12,7 @@ pytables
 html5lib
 lxml
 matplotlib
+Jinja2==2.8
```

Comment:
add this in requirements_all

and in install.rst (also need a test that gives a nice message if it's not installed)
eg df.style should raise